rabbitmq/tests/test_openmetrics.py

```diff
@@ -154,7 +178,7 @@ def mock_http_responses(url, **_params):
             ],
             id="two metric families",
         ),
-        pytest.param('per-object', DEFAULT_OPENMETRICS.difference(AGGREGATED_ONLY_METRICS), id='per-object'),
+        # pytest.param('per-object', DEFAULT_OPENMETRICS.difference(AGGREGATED_ONLY_METRICS), id='per-object'),
```

Comment:
Should we remove the commented out param? Line 108 is also commented out

Reply:
Yes! 😂 I forgot to delete those